src/Tools/dotnet-user-jwts/src/Helpers/DevJwtCliHelpers.cs

```diff
@@ -32,7 +32,7 @@ public static string GetProject(string projectPath = null)
             return projectPath;
         }
 
-        var csprojFiles = Directory.EnumerateFileSystemEntries(Directory.GetCurrentDirectory(), "*.*proj", SearchOption.TopDirectoryOnly)
+        var csprojFiles = Directory.EnumerateFileSystemEntries(Directory.GetCurrentDirectory(), "*.*proj", SearchOption.AllDirectories)
```

Comment:
Is this guaranteed to find the top most `*proj`? Like if my project was `b.csproj` but had a folder `a/` with an `a.csproj`, that wouldn't be used instead?

Reply:
The current behavior will throw an error for the multiple project scenario as it only supports looking for a single project file within a sub-directory. We can do something like use the first project file it finds but I'm not sure that's the best option. It feels more sensible to force the user to be explicit if we cannot find something.

Reply:
We should likely just match whatever the behavior of `user-secrets` is yes?

Reply:
`user-secrets` only searches the current directory if we wanted to stick to that.